app/pages/sustainabilityreports/page.tsx

```diff
@@ -0,0 +1,147 @@
+'use client';
+import { useState } from 'react';
+import Title from '../../components/Title';
+import { Input, Button, Typography } from '@material-tailwind/react';
+import { ArrowUpTrayIcon } from '@heroicons/react/24/solid';
+import { DefaultPagination } from '../../components/pagination';
+
+function Page() {
+  const TABLE_HEAD = [
+    'Company',
+    'References',
+    'Scope 1 CO2 emissions',
+    'Scope 2 CO2 emissions (location-based)',
+    'Potential financial impact figure',
+    'Action',
+  ];
+
+  const TABLE_ROWS = [
+    {
+      name: 'John Michael',
+      references: 'CDP Link',
+      scope1: 'scope1',
+      scope2: 'scope2',
+      potential: 'Potential',
+      action: 'Share',
+    },
+    {
+      name: 'Alexa Liras',
+      references: 'CDP Link',
+      scope1: 'scope1',
+      scope2: 'scope2',
+      potential: 'Potential',
+      action: 'Share',
+    },
+    {
+      name: 'Laurent Perrier',
+      references: 'CDP Link',
+      scope1: 'scope1',
+      scope2: 'scope2',
+      potential: 'Potential',
+      action: 'Share',
+    },
+    {
+      name: 'Michael Levi',
+      references: 'CDP Link',
+      scope1: 'scope1',
+      scope2: 'scope2',
+      potential: 'Potential',
+      action: 'Share',
+    },
+    {
+      name: 'Richard Gran',
+      references: 'CDP Link',
+      scope1: 'scope1',
+      scope2: 'scope2',
+      potential: 'Potential',
+      action: 'Share',
+    },
+  ];
+  const [Company, setCompany] = useState('');
+  const onChange = ({ target }) => setCompany(target.value);
+  return (
+    <div className="w-full h-full flex flex-col">
+      <Title label="CDP Sustainability Reports 2023 - Fortune 500" />
+
+      <div className="flex w-full gap-6 mt-8">
+        <div className="flex w-3/4 gap-6">
+          <Input label="Company Name" className="w-1/4" />
+          <Input label="Start Date" className="w-1/4" />
+          <Input label="End Date" className="w-1/4" />
+        </div>
+        <Button>Search</Button>
+      </div>
+
+      <div className="mt-8 gap-6 flex">
+        <Button>Add New Event</Button>
+        <Button>Add New Company</Button>
+        <Button className="flex item-center">
+          <ArrowUpTrayIcon className="w-4 mr-2" />
+          Browse Files to upload
+        </Button>
+      </div>
+
+      <div className="flex flex-col mt-8">
```

Comment:
Please handle overflow for adding more scopes

Reply:
done

Reply:
done